pkgs/by-name/lu/lubelog/package.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+}:
+
+buildDotnetModule {
+  pname = "lubelog";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = "hargata";
+    repo = "lubelog";
+    rev = "331499461a66dcba3dea2414c23bd4de902cd337";
+    hash = "sha256-uqS3Z7eRj3hRUperzCBOMJsAUbSftu06VVPTWJuXlQw=";
+  };
+
+  projectFile = "CarCareTracker.sln";
+  nugetDeps = ./deps.nix; # File generated with `nix-build -A package.passthru.fetch-deps`.
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_8_0;
+
+  makeWrapperArgs = [
+    "--set DOTNET_CONTENTROOT ${placeholder "out"}/lib/lubelog"
+  ];
+
+  executables = [ "CarCareTracker" ]; # This wraps "$out/lib/$pname/foo" to `$out/bin/foo`.
+
+  packNupkg = true; # This packs the project as "foo-0.1.nupkg" at `$out/share`.
+
+  meta = with lib; {
+    description = "Vehicle Service Records and Maintainence Tracker";
+    longDescription = ''
+      Self-Hosted, Open-Source, Unconventionally-Named Vehicle Maintenance Records and Fuel Mileage Tracker
+    '';
```

```suggestion
    description = "Vehicle service records and maintainence tracker";
    longDescription = ''
      Self-hosted, open-source, unconventionally-named vehicle maintenance records and fuel mileage tracker
    '';
```

- Description and longDescription does not differs a lot, but that's my personal opinion

---

Yeah, I'm just getting them from the website